bin/deploy

```diff
@@ -90,6 +95,27 @@ am_I_sourced() {
   fi
 }
 
+get_tagged_env() {
+  eups_root=https://eups.lsst.codes/stack
+  env_version=$($CURL "${CURL_OPTS[@]}" -L "$eups_root/src/tags/$1.list" \
```

Comment:
I tried to take a look at https://eups.lsst.codes/stack/src/tags and it seems to be broken because clicking on any of the links takes you to a non-functioning link of things like https://eups.lsst.codes/stack/src/w_2021_25.list rather than the expected https://eups.lsst.codes/stack/src/tags/w_2021_25.list

Reply:
You need a trailing slash when viewing in the browser.